README.md

```diff
@@ -101,6 +129,9 @@ console.log(user.method()); // outputs 'hello'
 
 ### Name based injection (hints)
 
+Instead of using the type of the component (the name of the class) as identifier for injection it is also possible to
```

Comment:
This is really the type of the class, not the name. Therefore it is minifier save. Its important to get that right.